markdown/dbaas.md

```diff
@@ -252,25 +246,97 @@ Once we registered kubernetes cluster we can use it’s name to create DB Cluste
 
 ```bash
 curl -X POST "http://localhost/v1/management/DBaaS/PXCCluster/Create" \ 
-     -H "accept: application/json" -u “admin:admin” -H "Content-Type: application/json" \ 
-     -d "
-{ 
-        \"kubernetes_cluster_name\": \"my_cluster\", 
-        \"name\": \"my-cluster-1\", 
-        \"expose\": true, 
-        \"params\": { \"cluster_size\": 3, \"pxc\": { \"compute_resources\": { \"cpu_m\": 1000, \"memory_bytes\": 2000000000 }, \"disk_size\": 25000000000, \"image\": \"percona/percona-xtradb-cluster:8.0.25-15.1\" }, \"haproxy\": { \"compute_resources\": { \"cpu_m\": 1000, \"memory_bytes\": 2000000000 } } }
+     -H "accept: application/json" \
+     -H "authorization: Basic YWRtaW46YWRtaW4=" \
+     -H "Content-Type: application/json" \ 
+     -d "{ \"kubernetes_cluster_name\": \"my_cluster\", \"name\": \"my-cluster-1\", \"expose\": true, \"params\": { \"cluster_size\": 3, \"pxc\": { \"compute_resources\": { \"cpu_m\": 1000, \"memory_bytes\": 2000000000 }, \"disk_size\": 25000000000, \"image\": \"percona/percona-xtradb-cluster:8.0.25-15.1\" }, \"haproxy\": { \"compute_resources\": { \"cpu_m\": 1000, \"memory_bytes\": 2000000000 } } } }"
+```
+
+### Request parameters
+
+```
+{
+  "kubernetes_cluster_name": "string",
+  "name": "string",
+  "params": {
+    "cluster_size": 0,
+    "pxc": {
+      "image": "string",
+      "compute_resources": {
+        "cpu_m": 0,
+        "memory_bytes": "string"
+      },
+      "disk_size": "string"
+    },
+    "proxysql": {
+      "image": "string",
+      "compute_resources": {
+        "cpu_m": 0,
+        "memory_bytes": "string"
+      },
+      "disk_size": "string"
+    },
+    "haproxy": {
+      "image": "string",
+      "compute_resources": {
+        "cpu_m": 0,
+        "memory_bytes": "string"
+      }
+    }
+  },
+  "expose": true
 }
-"
 ```
 
+
+
+|Parameter|Description|Notes|
+|-----|-----|-----|
+|kubernetes_cluster_name|Kubernetes cluster name|Required|
+|name|PXC cluster name to create|name|
+|cluster_size|Cluster size|Default: 3|
+|image|Docker image name|Default is the recommended version from the Percona's version service|
+|compute_resources.cpu_m|CPU resources millis|Default: 1000|
+|mcompute_resources.memory_bytes|Max memory size in bytes|Default: 2 Gb|
+|disk_size|Max disk size for the PXC instance|Default: 25 Gb|
```

Comment:
we need to figure out what is transfer here. I think it is Gi 
https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/#Quantity
https://physics.nist.gov/cuu/Units/binary.html

https://www.percona.com/doc/kubernetes-operator-for-pxc/operator.html#pxc-volumespec-resources-requests-storage

@BupycHuk what actually we have in API ?

Reply:
we pass G, not Gi at the moment. We can update FE and BE to use Gi instead of G.a

Reply:
we need to adjust it everywhere in a same manner (UI, API and Docs), also that it will be in sync with operator's doc.